### PR TITLE
Add pre-defined cohorts in responsibleaidashboard-diabetes-decision-making.ipynb

### PR DESCRIPTION
**Pull request by gaugup**
## Description

Add pre-defined cohorts in responsibleaidashboard-diabetes-decision-making.ipynb and related e2e tests

## Areas changed

npm packages changed:

- [ ] responsibleai/causality
- [ ] responsibleai/core-ui
- [ ] responsibleai/counterfactuals
- [ ] responsibleai/dataset-explorer
- [ ] responsibleai/fairness
- [ ] responsibleai/interpret
- [ ] responsibleai/localization
- [ ] responsibleai/mlchartlib
- [ ] responsibleai/model-assessment

Python packages changed:

- [ ] raiwidgets
- [ ] responsibleai
- [ ] erroranalysis
- [ ] rai_core_flask

## Tests

- [ ] No new tests required.
- [ ] New tests for the added feature are part of this PR.
- [ ] I validated the changes manually.

## Documentation:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by vinuthakaranth — APPROVED** [review]

**Review by imatiach-msft — APPROVED** [review]
only comment is we may want to do a release first before merging this so notebook will work with package on pypi

Yes, would not merge this until we have a release to pypi

File: notebooks/responsibleaidashboard/responsibleaidashboard-diabetes-decision-making.ipynb
Comment (romanlutz) [inline notebooks/responsibleaidashboard/responsibleaidashboard-diabetes-decision-making.ipynb:233]: typo continuous

File: notebooks/responsibleaidashboard/responsibleaidashboard-diabetes-decision-making.ipynb
Comment (romanlutz) [inline notebooks/responsibleaidashboard/responsibleaidashboard-diabetes-decision-making.ipynb:404]: is this a new thing? I've never seen these IDs before. Do they go away if you do the clear output command?

Comment (gaugup) [reply notebooks/responsibleaidashboard/responsibleaidashboard-diabetes-decision-making.ipynb:404]: Not sure why these got update. I am not sure how to correct these.

File: notebooks/responsibleaidashboard/responsibleaidashboard-diabetes-decision-making.ipynb
Comment (gaugup) [reply notebooks/responsibleaidashboard/responsibleaidashboard-diabetes-decision-making.ipynb]: fixed.

**Pull request merged**
